tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,161 @@
+
+
+def test_paddle_autolog_logs_default_params(paddle_model):
+    _, run = paddle_model
+    data = run.data
+    assert "optimizer_name" in data.params
+
+
+def test_paddle_autolog_logs_expected_data(paddle_model):
+    _, run = paddle_model
+    data = run.data
+
+    # Checking if metrics are logged
+    client = mlflow.tracking.MlflowClient()
+    for metric_key in ["batch_size", "loss", "step", "eval_batch_size", "eval_loss", "eval_step"]:
+        assert metric_key in run.data.metrics
+        metric_history = client.get_metric_history(run.info.run_id, metric_key)
+        assert len(metric_history) == NUM_EPOCHS
+
+    # Testing optimizer parameters are logged
+    assert "optimizer_name" in data.params
+    assert data.params["optimizer_name"] == "Adam"
+
+    # Testing learning rate are logged
+    assert "learning_rate" in data.params
+    assert float(data.params["learning_rate"]) == 1e-2
+
+    # Testing model_summary.txt is saved
+    client = mlflow.tracking.MlflowClient()
+    artifacts = client.list_artifacts(run.info.run_id)
+    artifacts = map(lambda x: x.path, artifacts)
+    assert "model_summary.txt" in artifacts
+
+
+def test_paddle_autolog_persists_manually_created_run(dataset, test_model):
+    with mlflow.start_run() as manual_run:
+        mlflow.paddle.autolog()
+
+        train_dataset, eval_dataset = dataset
+
+        test_model.fit(train_dataset, eval_dataset, epochs=NUM_EPOCHS, batch_size=8, verbose=1)
+        assert mlflow.active_run() is not None
+        assert mlflow.active_run().info.run_id == manual_run.info.run_id
+
+
+def test_paddle_autolog_ends_auto_created_run(paddle_model):  # pylint: disable=unused-argument
+    assert mlflow.active_run() is None
+
+
+@pytest.fixture
+def paddle_model_with_early_stopping(patience, dataset, test_model):
+    train_dataset, eval_dataset = dataset
+
+    callbacks_earlystopping = paddle.callbacks.EarlyStopping(
+        "loss",
+        mode="min",
+        patience=patience,
+        verbose=2,
+        min_delta=0,
+        baseline=None,
+        save_best_model=True,
+    )
+
+    mlflow.paddle.autolog()
+    test_model.fit(
+        train_dataset,
+        eval_dataset,
+        epochs=NUM_EPOCHS,
+        batch_size=8,
+        verbose=1,
+        callbacks=[callbacks_earlystopping],
+    )
+
+    client = mlflow.tracking.MlflowClient()
+    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
+    return test_model, run
+
+
+@pytest.mark.parametrize("patience", [1])
+def test_paddle_early_stop_params_logged(paddle_model_with_early_stopping, patience):
+    _, run = paddle_model_with_early_stopping
+    data = run.data
+    assert "monitor" in data.params
+    assert "patience" in data.params
+    assert float(data.params["patience"]) == patience
+    assert "min_delta" in data.params
+    assert "stopped_epoch" in data.params
+
+
+def test_paddle_autolog_non_early_stop_callback_does_not_log(paddle_model):
```

Comment:
Can you check the test names? For example, the name of this test doesn't match what's checked in this test.

Reply:
Done~